test/attachments_test.rb

```diff
@@ -66,8 +66,8 @@ class ActiveStorage::AttachmentsTest < ActiveSupport::TestCase
 
   test "attach new blobs" do
     @user.highlights.attach(
-      { io: StringIO.new("STUFF"), filename: "town.jpg", content_type: "image/jpg" }, 
-      { io: StringIO.new("IT"), filename: "country.jpg", content_type: "image/jpg" })
+      { io: StringIO.new("STUFF"), filename: "town.jpg", content_type: "image/jpg" },
+      io: StringIO.new("IT"), filename: "country.jpg", content_type: "image/jpg")
```

Comment:
Actually the rule has a `context_dependent` config which enforces braces in this case:

```
# bad
some_method(x, y, {a: 1, b: 2})
some_method(x, y, {a: 1, b: 2}, a: 1, b: 2)

# good
some_method(x, y, a: 1, b: 2)
some_method(x, y, {a: 1, b: 2}, {a: 1, b: 2})
```